bloodpet/ev-onboarding-exercise
# Create frontend app

# Tickets

- https://phunts.atlassian.net/browse/EVOBE-112
- https://phunts.atlassian.net/browse/EVOBE-113
- https://phunts.atlassian.net/browse/EVOBE-114
- https://phunts.atlassian.net/browse/EVOBE-115
- https://phunts.atlassian.net/browse/EVOBE-116
- https://phunts.atlassian.net/browse/EVOBE-117
- https://phunts.atlassian.net/browse/EVOBE-118

# Changes

- Create react app
- Create employee
- Edit employee
- Delete employee
- View employee
- Employee list
- Add to docker-compose

# Dependencies

- docker
- docker-compose

# Note

I did everything in one page.

# Test plan

Pull branch

1. Start container: `docker-compose up frontend`
2. Open in browser: http://localhost:5000/
3. Add employee by clicking "Add employee", edit & save
4. See new employee in the list
5. See the new employee by clicking their name
6. Edit the employee by clicking "Edit", edit & save
7. Check the updated employee
